dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -16,8 +16,50 @@
 
 # ACME configuration
 
-#configParm1=Configuration parameter 1 for ACME
-#configParm1.desc=Description for parameter 1
+#directoryURI=ACME server directory URI
+#directoryURI.desc=The URI to the ACME server's directory object.
 
-#configParm2=Configuration parameter 2 for ACME
-#configParm2.desc=Description for parameter 2
+#domain=Domain name
+#domain.desc=A domain name to request a certificate for.
+
+#validFor=Valid for
+#validFor.desc=The duration of time the certificate signing request will request the certificate to be valid for. The default is defined by the CA server.
+
+#country=Country
+#country.desc=Country name to use for the certificate.
+
+#state=State or province
+#state.desc=State or province name to use for the certificate.
+
+#locality=Locality
+#locality.desc=Locality name to use for the certificate.
+
+#organization=Organization
+#organization.desc=Organization name to use for the certificate.
+
+#organizationalUnit=Organizational unit
+#organizationalUnit.desc=Organizational unit name to use for the certificate.
+
+#acceptTermsOfService=Accept Terms of Service
+#acceptTermsOfService.desc=Indicates whether the user accepts the terms of service for the ACME CA. If true, the terms of service will be automatically accepted. If false, and the ACME CA issues terms of service, the request for a certificate will fail.  
```

Comment:
**change**
If true, the terms of service will be automatically accepted. If false, and the ACME CA issues terms of service, the request for a certificate will fail. 
**to**
If true, the terms of service is automatically accepted. If false, and the ACME CA issues terms of service, the request for a certificate fails.

Reply:
Made the requested changes.